libraries/cli/include/cli/default_config.hpp

```diff
@@ -134,6 +134,10 @@ const char *default_json = R"foo({
           {
             "name": "SUMMARY",
             "verbosity": "INFO"
+          },
+          {
+            "name": "SORT_MGR",
+            "verbosity": "DEBUG"
```

Comment:
this change does not seem to be related to genesis balances hardfork ?

Reply:
I added it to get some logs during PRnet testing, so I will just remove it after